backend/pkg/utils/utils.go

```diff
@@ -26,3 +27,27 @@ func DedupeSliceOrdered[T comparable](input []T) []T {
 	}
 	return output
 }
+
+func DedupeSlice[T comparable](input []T) []T {
```

Comment:
FWIW, I think we can actualy remove this function entirely and just use [slices.Compact](https://pkg.go.dev/slices#Compact)